### PR TITLE
Added more tooltips to game buttons

### PR DESCRIPTION
**Pull request by Kasterisk**
**Brief Description of What This PR Does**

Found that in old versions tooltips were quite abundant but some of them seems to be lost now. This brought back some of those tooltips, no reason not to have them now. Tooltips readded for: some of the menu buttons, pause menu buttons and editor button. 

Also tweaked the version number and store user tag label's font and size.

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: locale/en.po [hunk 1]
```diff
@@ -1595,48 +1607,52 @@ msgstr "Extras"
 msgid "CREDITS"
 msgstr "Credits"
 
-#: ../src/general/MainMenu.tscn:257
+#: ../src/general/MainMenu.tscn:254 ../src/general/PauseMenu.tscn:236
+msgid "QUIT_BUTTON_TOOLTIP"
+msgstr "Exit to desktop"
```
Comment (hhyyrylainen) [inline locale/en.po:1612]: We may have other platforms in the future, so the term "desktop" should be avoided here. Maybe "Exit the game" would be a good enough alternative?

File: locale/en.po [hunk 2]
```diff
@@ -1986,33 +2002,54 @@ msgstr ""
...
+msgstr "Return to the game"
+
+#: ../src/general/PauseMenu.tscn:180
 msgid "RESUME"
 msgstr "Resume"
 
-#: ../src/general/PauseMenu.tscn:185
+#: ../src/general/PauseMenu.tscn:187
+msgid "SAVE_GAME_BUTTON_TOOLTIP"
+msgstr "Open the save dialog to save the game"
```
Comment (hhyyrylainen) [inline locale/en.po:2015]:
```suggestion
msgstr "Open the save menu to save the game"
```

File: locale/en.po [hunk 3]
```diff
@@ -2964,87 +3001,87 @@ msgstr "Can't use spawn enemy cheat because this patch does not contain any enem
...
 
-#: ../src/microbe_stage/MicrobeStage.tscn:1823
+#: ../src/microbe_stage/MicrobeStage.tscn:1822
 msgid "POPULATION_CAPITAL"
 msgstr "POPULATION:"
 
-#: ../src/microbe_stage/MicrobeStage.tscn:2121
+#: ../src/microbe_stage/MicrobeStage.tscn:1969
+msgid "EDITOR_BUTTON_TOOLTIP"
+msgstr "Enter the editor and mutate your cell"
```
Comment (hhyyrylainen) [inline locale/en.po:3082]:
```suggestion
msgstr "Enter the editor and modify your species"
```
to be more general and not need to update this for early multicellular

File: locale/en.po [hunk 4]
```diff
@@ -1566,16 +1566,28 @@ msgstr "Glucose concentrations have drastically dropped!"
 msgid "STORE_LOGGED_IN_AS"
 msgstr "Logged in as: {0}"
 
-#: ../src/general/MainMenu.tscn:178
+#: ../src/general/MainMenu.tscn:173
+msgid "NEW_GAME_BUTTON_TOOLTIP"
+msgstr "Enter the microbe stage!"
```
[the remark this answers is not in the document]
Comment (Kasterisk) [reply locale/en.po:1571]: That's not descriptive of what the new game actually does. I thought new game always starts at the microbe stage and starting at any other stage is not allowed?

[the remark this answers is not in the document]
Comment (Kasterisk) [reply locale/en.po:1571]: Oh yeah, for that that I guess this tooltip will have to be relegated to the confirm button in that new game options screen.

**Review by hhyyrylainen — APPROVED** [review]
I think all of my concerns were addressed with the texts.

**Pull request merged**